src/utilities/intel/getWeatherbyLatLng.js

```diff
@@ -0,0 +1,49 @@
+export default async function getWeatherbyLatLng(lat, lng) {
+  // Check if lat/lng have been set
+  if(lat && lng){
+
+    const time = new Date(new Date().setFullYear(new Date().getFullYear() -1 ));
+    let timestamp = Math.round(time/1000);
+
+    // Set API link and infuse with data
+    const api = `https://cors-anywhere.herokuapp.com/https://api.darksky.net/forecast/
+${process.env.REACT_APP_DARKSKY_APIKEY}/
```

Comment:
I think there are some indents missing. You should add them.

Reply:
Wrong. If indents are present, the url contains those indents.